packages/vm/examples/run-transactions-complete/index.ts

```diff
@@ -18,21 +17,19 @@ async function main() {
   console.log('Sender address: ', bufferToHex(address))
 
   // create a new account
-  const account = new Account({
-    balance: 100e18,
-  })
+  const account = new Account(new BN(0), new BN(100).pow(new BN(18)))
```

Comment:
This is actually not getting more readable using the constructor directly compared to the old version (same above), do you might want to switch to `Account.fromAccountData()` instead?

Reply:
👍 done in https://github.com/ethereumjs/ethereumjs-vm/pull/911/commits/588912a2b1a5bf8b88f07613d7fec9338ac6ff51